### ISSUE TITLE
Tags des discussions (flairs)

### ISSUE THREAD

**Issue by mgkprod** [4sucres/board#71]
https://github.com/4sucres/board/issues/9#issuecomment-493775581

L'implémentation des flairs sur 4S permettrait de reproduire un équivalent "sous-catégorie" sans forcément pénaliser une discussion ou une autre (un post sur #hobbies reste un sujet lié aux hobbies)

Avant tout, il faut définir les tags que l'on souhaite rendre disponible par catégories (voir avec les membres)

**Comment by hawezo** [comment]
Absolument. Dans un premier temps, il faudrait en effet définir la liste des tags par catégorie. 

Ensuite, il faudra créer une API qui prendra comme paramètre le titre et la catégorie et retournera une liste de flairs potentiels. 

C'est assez facile à faire, il faudra juste penser aux mots-clés. Bonne idée :wink:

**Comment by hawezo** [comment]
@mgkprod Le titre de ce ticket devrait être changé en quelque chose comme "Suggestion automatique de flairs"
